ReactiveXComponentTest/UnitTests/WebSocketUnitTests/WebsocketMessageHelperTest.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using NFluent;
+using NUnit.Framework;
+using ReactiveXComponent.Common;
+using ReactiveXComponent.WebSocket;
+
+namespace ReactiveXComponentTest.UnitTests.WebSocketUnitTests
```

Comment:
Shouldn't this file be removed as the same one was created under _ReactiveXComponentTest\WebSocket_ ?